### PR TITLE
Add target FLMO: GRAVITYF7

### PR DESCRIPTION
**Pull request by luka177**
Based on #6005 but with cmake instead of mk.

### CODE REVIEW

File: src/main/target/GRAVITYF7/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+target_stm32f722xe(GRAVITYF7)
```
Comment (mmosca) [inline src/main/target/GRAVITYF7/CMakeLists.txt]: Can you add SKIP_RELEASES?

Comment (luka177) [reply src/main/target/GRAVITYF7/CMakeLists.txt]: done

**Comment by mmosca** [comment]
There are still some conflicts. Can you fix those and update the target.c entries to use the new TIM_USE_OUTPUT_AUTO?

**Comment by mmosca** [comment]
Stale.

**Pull request closed without merging**